packages/sui-widget-embedder/src-react/render.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
👀

Reply:
It's in purpose! I had problems while linking the package with errors on the compiled one. This will ensure the "magic" comment of eslint is kept after preparing the react files so the sui-bundler doesn't give problems. Maybe that happens only on linking the sui-widget-embedder, I could remove them and try again with the published package.